test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/function/ToLongBiFunctionTest.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.function
+
+import org.junit.Assert._
+import org.junit.Test
+
+import java.util.function._
+
+class ToLongBiFunctionTest {
+  @Test def applyAsLong(): Unit = {
+    val op = new ToLongBiFunction[String, String] {
+      override def applyAsLong(t: String, u: String): Long = t.toLong * u.toLong
+    }
+    assertEquals(op.applyAsLong("11111111", "11111111"), 123456787654321L)
```

Comment:
Take two different strings? Just for good measure?

Reply:
Fixed https://github.com/scala-js/scala-js/pull/4365/commits/792d6c735a519163b83d6f39ec9dbbbcfa7aad24